graylog2-server/src/main/java/org/graylog2/bindings/providers/EsNodeProvider.java

```diff
@@ -48,43 +47,45 @@ public EsNodeProvider(ElasticsearchConfiguration configuration) {
     }
 
     @Override
-    @Singleton
```

Comment:
AFAIK the annotation is being ignored at the `Provider#get()` method. Only methods annotated with `@Provides` and `@Singleton` inside a Guice module will have the desired effect.

Additionally `EsNodeProvider` is explicitly being bound as eager singleton in `ServerBindings`, so the `@Singleton` annotation in `EsNodeProvider` is more for documentation purposes.